src/pages/admin/faq/components/faq-component/FaqComponent.scss

```
@@ -102,50 +102,48 @@
-                    background: url('~@/assets/icons/delete-pressed.svg');
+                    path {
+                        fill: colors.$blue-900;
+                    }
                 }
 
                 &:disabled {
-                    background: url('~@/assets/icons/delete-disabled.svg');
+                    path {
+                        fill: colors.$grey-600;
+                    }
                     cursor: not-allowed;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)